linkerd/app/outbound/src/http/concrete/tests.rs

```diff
@@ -0,0 +1,152 @@
+use super::*;
+use crate::test_util::*;
+use linkerd_app_core::{
+    svc::{http::balance::EwmaConfig, ServiceExt},
+    svc::{NewService, Service},
+    trace,
+};
+use linkerd_proxy_client_policy as policy;
+use std::{net::SocketAddr, num::NonZeroU16, sync::Arc};
+use tokio::time;
+
+#[tokio::test(flavor = "current_thread")]
+async fn gauges_endpoints() {
+    let _trace = trace::test::trace_init();
+    let (rt, _shutdown) = runtime();
+    let outbound = Outbound::new(default_config(), rt);
+
+    let addr = format!("mysvc.myns.svc.cluster.local:80")
```

Comment:
clippy would prefer you use 
```suggestion
    let addr = "mysvc.myns.svc.cluster.local:80"
        .to_string()
```